src/types.ts

```diff
@@ -80,6 +80,7 @@ export interface IHostConfig {
   offrampWebhookV3Url?: TWebhookStatusUrl;
   useSendCryptoCallback?: boolean;
   paymentMethodType?: PaymentMethodType;
+  customExitButton?: boolean;
```

Comment:
same comment as in the app PR - please change the param name

Reply:
@macbem done 23b19dcbf8e79901b16caecb3e2e75c45d5246dd